src/py/mat3ra/made/tools/modify.py

```diff
@@ -323,3 +319,42 @@ def condition(coordinate):
     return filter_by_condition_on_coordinates(
         material, condition, use_cartesian_coordinates=use_cartesian_coordinates, invert_selection=invert_selection
     )
+
+
+def add_vacuum(material: Material, vacuum: float = 5.0) -> Material:
+    """
+    Add vacuum to the top of the material.
+
+    Args:
+        material (Material): The material object to add vacuum to.
+        vacuum (float): The thickness of the vacuum to add in angstroms.
+
+    Returns:
+        Material: The material object with vacuum added.
+    """
+    new_material_atoms = to_ase(material)
+    ase_add_vacuum(new_material_atoms, vacuum)
+    new_material = Material(from_ase(new_material_atoms))
+    return new_material
+
+
+def set_vacuum(material: Material, vacuum: float = 5.0) -> Material:
+    """
+    Set the vacuum thickness of the material along the c-axis.
+    This function first strips any existing vacuum and then sets the new vacuum thickness specified.
+
+    Args:
+        material (Material): The material object to set the vacuum thickness.
+        vacuum (float): The total thickness of the vacuum in angstroms to set above the material.
+
+    Returns:
+        Material: The material object with the vacuum thickness set.
+    """
+    atoms = to_ase(translate_to_bottom(material))
+    new_c = max(atoms.positions[:, 2]) + vacuum
```

Comment:
With zero vacuum this will lead to the first and last layer being at the same coordinate, it seems no?

Reply:
Yes, this is somewhat dangerous operation. You get what you ask for. Because there's no general way to set a distance correctly: covalent radius of last atoms? average interlayer distance in the material in Z? Distance between A and B materials in interface? None will fit all the possibilities

Reply:
Ok. The covalent radius of the topmost atom sounds like a good middle ground. We need to import periodic_table into py for that